README.md

```diff
@@ -76,7 +76,8 @@ Please subscribe and '+1' the videos if you find them useful.
 ## Configuration Tool
 
 To configure Cleanflight you should use the [Cleanflight-configurator GUI tool](https://chrome.google.com/webstore/detail/cleanflight-configurator/enacoimjcgeinfnnnpajinjgmkahmfgb
-) (Windows/OSX/Linux) that can be found here:
+) (Windows/OSX/Linux) that can be found [here](https://chrome.google.com/webstore/detail/cleanflight-configurator/enacoimjcgeinfnnnpajinjgmkahmfgb
+).
```

Comment:
we don't need the link twice.  also 'here' as bad SEO too.  since the link title does not match the content. :smile: